lib/srte.h

```diff
@@ -19,6 +19,23 @@ enum zebra_sr_policy_status {
 	ZEBRA_SR_POLICY_DOWN,
 };
 
+/* SR types. */
+enum sr_types_t {
+	ZEBRA_SR_LSP_NONE = 0,	/* No LSP. */
+	ZEBRA_SR_LSP_SRTE = 1,	/* SR-TE LSP */
+	ZEBRA_SR_SRV6_SRTE = 2, /* SRv6 SID List*/
+};
+
+static inline enum lsp_types_t lsp_type_from_sr_type(int sr_type)
```

Comment:
Shouldn't this pass in a `enum sr_types..` instead of an int? Let's let the compiler help us please

Reply:
fixed

Reply:
Please remove the default case here.

Reply:
fixed